test/e2e/common/setup.go

```diff
@@ -228,18 +233,22 @@ func InstallOrUpgradeFederationControllers(spireEnabled bool) resource.SetupFn {
 			if err != nil {
 				return err
 			}
+			remoteAddr := gatewayIP
+			if opts.SetRemoteDNSName {
+				remoteAddr = fmt.Sprintf("ingress.%s", remoteClusterName)
+			}
```

Comment:
Minor readability suggestion avoiding variable reassignment:
```suggestion
			var remoteAddr string
			if opts.SetRemoteDNSName {
				remoteAddr = fmt.Sprintf("ingress.%s", remoteClusterName)
			} else {
			    remoteAddr = gatewayIP
			}
```

Reply:
Done :)